compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/window/ComposeWindow.uikit.kt

```diff
@@ -607,15 +607,32 @@ internal actual class ComposeWindow : UIViewController {
 
             override fun pointInside(point: CValue<CGPoint>, event: UIEvent?): Boolean =
                 point.useContents {
-                    val hitsInteropView = attachedComposeContext?.scene?.mainOwner?.hitInteropView(
-                        pointerPosition = Offset(
-                            (x * density.density).toFloat(),
-                            (y * density.density).toFloat()
-                        ),
-                        isTouchEvent = true,
-                    ) ?: false
-
-                    !hitsInteropView
+                    val position = Offset(
+                        (x * density.density).toFloat(),
+                        (y * density.density).toFloat()
+                    )
+
+                    // TODO: temporary solution copying control flow from [ComposeScene.processPress]
+                    // there is no fastReverseAny in Kotlin
+                    for (i in scene.owners.size - 1 downTo 0) {
```

Comment:
Let's create `internal fun ComposeScene.hitInteropView` and move the code there. The overall design of ComposeScene is that it should hide `Owner`'s in itself.

Reply:
Done